src/hotspot/share/opto/graphKit.hpp

```diff
@@ -790,9 +790,6 @@ class GraphKit : public Phase {
   // rounding for strict double precision conformance
```

Comment:
Isn't the reference to "strict" here and in the cpp file no longer applicable?

Reply:
> Isn't the reference to "strict" here and in the cpp file no longer applicable?

```
  // Advertise here if the CPU requires explicit rounding operations to implement strictfp mode.
#ifdef _LP64
  static const bool strict_fp_requires_explicit_rounding = false;
#else
  static const bool strict_fp_requires_explicit_rounding = true;
#endif
```
It seems that only x86_32 needs explicit rounding for strict_fp operations.

Reply:
Yup, "strict" is still meaningful on x86_32 FPU. It might be confusing due to fact that JDK 17 is now strictfp-by-default, but rounding to support strictfp is still needed for some awkward arches.

Reply:
My point was that the now deleted function claimed it was the rounding function for non-strict, while the function that has been kept is described as being for strict. But now there is only one function that has to cater for both so using "strict" in its description seems inappropriate.